src/ReflectionProperty.php

```diff
@@ -60,7 +60,8 @@ public function __construct(
         Property $propertyType = null,
         PropertyProperty $propertyNode = null
     ) {
-        $this->className    = $className;
+        $this->className    = is_object($className) ? get_class($className) : ltrim($className, '\\');
```

Comment:
Does this class even support passing an object in `$className` parameter? I'm suspecting it doesn't, because parameter name suggests that. Yeah, I know that PHP built-in reflection supports that.

If it doesn't, then do `ltrim` all the time.